packages/api/bin/serve.js

```diff
@@ -300,7 +300,7 @@ async function eraseDynamoTables(stackName, systemBucket) {
 
   // Remove all data from tables
   const providerModel = new models.Provider();
-  const collectionModel = new models.Collection();
+  const collec tionModel = new models.Collection();
```

Comment:
this was probably a typo, but I think it should be `collectionModel` ?

Reply:
Well *that* should fail a test.     :thinking face: